packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/controller.js

```diff
@@ -177,9 +177,12 @@ export default class CreateLinkedUserController {
       })
       .then((credential) => {
         this.userModel.createMode.credential = credential;
-
+        this.trackPage(`${TRACKING_CREATE_USER}-success`);
```

Comment:
It's not `this.atInternet.trackPage` ?

Reply:
I have defined a trackPage()  function is controller which in turns calls this.atInternet.trackPage().  I have defined to avoid code duplication.